api/alembic/versions/82cc8ffa75ce_update_kamloops_stations.py

```diff
@@ -0,0 +1,156 @@
+"""Update Kamloops stations
+
+Revision ID: 82cc8ffa75ce
+Revises: c525dbd0c37e
+Create Date: 2022-04-14 13:30:21.139779
+
+"""
+from typing import List, Optional
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm.session import Session
+
+# revision identifiers, used by Alembic.
+revision = '82cc8ffa75ce'
+down_revision = 'c525dbd0c37e'
+branch_labels = None
+depends_on = None
+
+# New station, pentiction
+west_kelowna_station_code = 1277
+
+# New station, vernon
+station_bay_2_station_code = 1359
+
+# Pentiction station, order to be adjusted
+mccuddy_station_code = 334
+
+# Vernon stations, order to be adjusted
+seymour_arm = 344
+salmon_arm = 346
+kettle_2 = 388
```

Comment:
Not related to this PR, but I'm thinking at some point in the future it would be handy if we updated the existing ordering numbers to be multiples of 10, so that we can easily insert new stations without having to do this shuffling dance every time.

Reply:
Nice, if you have a solution to make things easier, I'd recommend writing a dev task for it!

Reply:
When we have an admin interface, that issue will go away though - we'll have to do the shuffle in code.